### PR TITLE
Abstract css code / css to scss

### PR DESCRIPTION
**Pull request by amitg01**

### CODE REVIEW

**Comment by amitg01** [comment]
@n1c01a5 Please merge PR #44, so that I can rebase this branch with master and resolve the merge conflicts.

**Comment by amitg01** [comment]
Hi @n1c01a5 @bhavesh92pandey The changes for layout and other minor fixes are done. Here is a demo: https://www.loom.com/share/b4cde9b7c0a54b399be672ec08b3812e

**Comment by bhavesh92pandey** [comment]
Hi Amit, i saw the video but i think there are few changes still needs to be done. This we can discuss on the scheduled call

**Comment by amitg01** [comment]
Sure.

File: styles/globals.scss [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+.additionalClassForHeadMobile:hover,
+.active-accordion {
+  color: #444 !important;
+  background: #fff !important;
+}
+.additionalClassForContent {
+  background: #fff !important;
+  margin: 0 !important;
+  padding: 17px 0 !important;
+  line-height: 24px !important;
```
Comment (n1c01a5) [inline styles/globals.scss:232]: I am not big fan of these `!important` but ok.

**Review by n1c01a5 — CHANGES_REQUESTED** [review]
I left my review. See my comments.

**Pull request merged**